crates/bevy_ui/src/render/mod.rs

```diff
@@ -76,19 +78,28 @@ pub fn build_ui_render(app: &mut App) {
         .init_resource::<ExtractedUiNodes>()
         .init_resource::<DrawFunctions<TransparentUi>>()
         .add_render_command::<TransparentUi, DrawUi>()
+        .configure_sets(
+            ExtractSchedule,
+            (
+                RenderUiSystem::ExtractBackground,
+                RenderUiSystem::ExtractDecoration,
+                RenderUiSystem::ExtractImage,
+                RenderUiSystem::ExtractText,
```

Comment:
```suggestion
                RenderUiSystem::ExtractBackground,
                RenderUiSystem::ExtractImage,
                RenderUiSystem::ExtractDecoration,
                RenderUiSystem::ExtractText,
```

Borders should be drawn between images and text.

Reply:
My thinking was that images should be considered "foreground content", same as text. Reordering is easy though so I'll follow this suggestion for now.